chromadb/server/fastapi/__init__.py

```diff
@@ -762,7 +763,7 @@ def process_add(request: Request, raw_body: bytes) -> bool:
                 return self._api._add(
                     collection_id=_uuid(collection_id),
                     ids=add.ids,
-                    embeddings=add.embeddings,  # type: ignore
+                    embeddings=cast(Embeddings, [np.array(embedding) for embedding in add.embeddings] if add.embeddings else None),
```

Comment:
This is the reverse of `convert_np_embeddings_to_list` right? Maybe extract it as a function too.